api/core/splitter/fixed_text_splitter.py

```diff
@@ -66,7 +67,10 @@ def __init__(self, fixed_separator: str = "\n\n", separators: Optional[list[str]
     def split_text(self, text: str) -> list[str]:
         """Split incoming text and return chunks."""
         if self._fixed_separator:
-            chunks = text.split(self._fixed_separator)
+            if re.match(r'^reg\/.+\/$',self._fixed_separator):
```

Comment:
@Yeuoly Please take a look at this regex.

Reply:
Possible redos vulnerability